java/src/main/java/com/powsybl/dataframe/network/extensions/SecondaryVoltageControlDataframeProvider.java

```diff
@@ -0,0 +1,115 @@
+import com.powsybl.dataframe.network.ExtensionInformation;
+import com.powsybl.dataframe.network.NetworkDataframeMapper;
+import com.powsybl.dataframe.network.NetworkDataframeMapperBuilder;
+import com.powsybl.dataframe.network.adders.NetworkElementAdder;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.extensions.SecondaryVoltageControl;
+import com.powsybl.iidm.network.extensions.SecondaryVoltageControl.ControlZone;
+
+import java.util.*;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+/**
+ * @author Hugo Kulesza <hugo.kulesza@rte-france.com>
+ */
+@AutoService(NetworkExtensionDataframeProvider.class)
+public class SecondaryVoltageControlDataframeProvider implements NetworkExtensionDataframeProvider {
+
+    @Override
+    public String getExtensionName() {
+        return SecondaryVoltageControl.NAME;
+    }
+
+    @Override
+    public ExtensionInformation getExtensionInformation() {
+        return new ExtensionInformation(SecondaryVoltageControl.NAME,
+                "Provides information about the secondary voltage control zones and units, in two distinct dataframes.",
+                "[dataframe \"zones\"] index : name (str), target_v (float), bus_ids (str) / [dataframe \"units\"] index : unit_id (str), participate (bool), zone_name (str)");
+    }
+
+    @Override
+    public List<Optional<String>> getExtensionTableNames() {
+        return List.of(Optional.of("zones"), Optional.of("units"));
+    }
+
+    private Stream<ControlZone> zonesStream(Network network) {
+        return network.getExtension(SecondaryVoltageControl.class)
+                .getControlZones().stream();
+    }
+
+    private Stream<ControlUnitWithZone> unitsStream(Network network) {
+        List<ControlUnitWithZone> units = new ArrayList<>();
+        network.getExtension(SecondaryVoltageControl.class)
+                .getControlZones()
+                .forEach(zone -> {
+                    units.addAll(zone.getControlUnits()
+                            .stream()
+                            .map(unit -> {
+                                return new ControlUnitWithZone(unit, zone.getName());
+                            })
+                            .collect(toList())
+                    );
+                });
+        return units.stream();
+    }
+
+    @Override
+    public Map<Optional<String>, NetworkDataframeMapper> createMappers() {
+        Map<Optional<String>, NetworkDataframeMapper> mappers = new HashMap<>();
+        mappers.put(Optional.of("zones"),
+                NetworkDataframeMapperBuilder.ofStream(this::zonesStream)
+                        .stringsIndex("name", ControlZone::getName)
+                        .doubles("target_v", zone -> zone.getPilotPoint().getTargetV(), (zone, v) -> zone.getPilotPoint().setTargetV(v))
+                        .strings("bus_ids", zone -> zone.getPilotPoint().getBusbarSectionsOrBusesIds().get(0)) // TODO : parser liste ids
```

Comment:
indeed there is a small todo here :)
just joining the strings using a "," should be OK